cms/static/js/spec/views/modals/move_xblock_modal_spec.js

```diff
@@ -0,0 +1,63 @@
+define(['jquery', 'underscore', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers',
+        'common/js/spec_helpers/template_helpers', 'js/views/modals/move_xblock_modal', 'js/models/xblock_info'],
+    function($, _, AjaxHelpers, TemplateHelpers, MoveXBlockModal, XBlockInfo) {
+        'use strict';
+        describe('MoveXBlockModal', function() {
+            var modal,
+                showModal,
+                DISPLAY_NAME = 'HTML 101',
+                OUTLINE_URL = '/course/cid?format=concise',
+                ANCESTORS_URL = '/xblock/USAGE_ID?fields=ancestorInfo';
+
+            showModal = function() {
+                modal = new MoveXBlockModal({
+                    sourceXBlockInfo: new XBlockInfo({
+                        id: 'USAGE_ID',
+                        display_name: DISPLAY_NAME,
+                        category: 'html'
+                    }),
+                    XBlockUrlRoot: '/xblock',
+                    outlineURL: OUTLINE_URL,
+                    XBlockAncestorInfoUrl: ANCESTORS_URL
+
+                });
+                modal.show();
+            };
+
+            beforeEach(function() {
+                TemplateHelpers.installTemplates([
+                    'basic-modal',
+                    'modal-button',
+                    'move-xblock-modal'
+                ]);
+            });
+
+            afterEach(function() {
+                modal.hide();
+            });
+
+            it('rendered as expected', function() {
+                showModal();
+                expect(
+                    modal.$el.find('.modal-header .title').contents().get(0).nodeValue.trim()
```

Comment:
```HTML
<div class="modal-header">
    <h2 id="modal-window-title" class="title modal-window-title">
        Move: Blank HTML Page
        <span class="sr modal-sr-title">
            Choose a location to move your component to
        </span>
    </h2>
...
```

@mushtaqak @andy-armstrong This is how the modal header html looks like. I want to extract `Move: Blank HTML Page` only and `.contents().get(0).nodeValue` is the only way to do this. `.text()` will give the extra text which i don't want.